update_data.R

```diff
@@ -0,0 +1,313 @@
+                                               `Miss.organisation_1` = "Other", `Miss.organisation_2` = "Other",`Miss.organisation_6` = "Other"))
+    plhdata_org_clean <- plhdata_org %>%
+      filter(Org != "Other")%>%
+      mutate(Org = factor(Org))
+    plhdata_org_clean <- plhdata_org_clean %>%
+      dplyr::filter(!is.na(app_version))
+    
+    # make vars numeric
+    plhdata_org_clean$rp.contact.field.user_age <- as.numeric(plhdata_org_clean$rp.contact.field.user_age)
+    plhdata_org_clean$rp.contact.field.household_adults <- as.numeric(plhdata_org_clean$rp.contact.field.household_adults)
+    plhdata_org_clean$rp.contact.field.household_teens <- as.numeric(plhdata_org_clean$rp.contact.field.household_teens)
+    plhdata_org_clean$rp.contact.field.household_babies <- as.numeric(plhdata_org_clean$rp.contact.field.household_babies)
+    plhdata_org_clean$rp.contact.field.household_children <- as.numeric(plhdata_org_clean$rp.contact.field.household_children)
+    plhdata_org_clean$rp.contact.field.w_1on1_diff_started_completed <- as.numeric(plhdata_org_clean$rp.contact.field.w_1on1_diff_started_completed)
+    plhdata_org_clean$rp.contact.field.parent_point_count_relax_w_self_care <- as.numeric(plhdata_org_clean$rp.contact.field.parent_point_count_relax_w_self_care)
+    plhdata_org_clean <- plhdata_org_clean %>%
+      mutate(across(ends_with("_completion_level"), ~as.numeric(.)))
+    plhdata_org_clean <- plhdata_org_clean %>%
+      mutate(across(starts_with("rp.contact.field.parent_point"), ~as.numeric(.)))
+    plhdata_org_clean$rp.contact.field.app_launch_count <- as.numeric(plhdata_org_clean$rp.contact.field.app_launch_count)
+    plhdata_org_clean$rp.contact.field.w_self_care_diff_started_completed <- as.numeric(plhdata_org_clean$rp.contact.field.w_self_care_diff_started_completed)
+    plhdata_org_clean$rp.contact.field.first_app_open <- as.Date(plhdata_org_clean$rp.contact.field.first_app_open)
+    plhdata_org_clean$hsqsclickedws1<-!is.na(plhdata_org_clean$rp.contact.field.click_hs_weekly_workshops_quick_start_w_self_care)
+    plhdata_org_clean$rp.contact.field.max_days_between_app_launches <- as.numeric(plhdata_org_clean$rp.contact.field.max_days_between_app_launches)
+    
+    # select variables 
+    var_names <- var_names[-length(var_names)]
+    plhdata_org_clean_dat <- plhdata_org_clean %>%
+      dplyr::select(c(var_names))
+    
+    # add timestamp
+    if (nrow(plhdata_org_clean_dat) > 0){
+      plhdata_org_clean_dat$R_timestamp <- date()
+      
+      # then we bind the old data with this updated data
+      plhdata_org_clean_dat2 <- bind_rows(old_data, plhdata_org_clean_dat) 
+      saveRDS(plhdata_org_clean_dat2, "plhdata_org_clean_dat.RDS")
+      
+    } else {
+      print("No update required")
+      saveRDS(plhdata_org, "plhdata_org_clean_dat.RDS")
+    }
+  }
+}
+
+update_data()
+
+# Data frame for first time --------------------------------------------------------------------------
```

Comment:
This bit of the script onwards is relevant to just create the data frame the first time.

@MargheritaPhilipp I have taken this bit of the script from the analysis file